ocs_ci/ocs/registry.py

```diff
@@ -364,6 +364,14 @@ def image_pull_and_push(
     """
     if config.DEPLOYMENT.get('disconnected'):
         mirror_image(image=image)
+
+    elif config.DEPLOYMENT.get('powervs'):
```

Comment:
Not sure if this works as we do not have deployment config for s390s

Reply:
Agreed, now verify the template `eap-cd-basic-s2i` is present in cluster if not pull supported openshift redis-ephemeral images